packages/hardhat/scripts/generateAccount.ts

```diff
@@ -1,42 +1,55 @@
 import { ethers } from "ethers";
 import { parse, stringify } from "envfile";
 import * as fs from "fs";
+import password from "@inquirer/password";
 
 const envFilePath = "./.env";
 
-/**
- * Generate a new random private key and write it to the .env file
- */
-const setNewEnvConfig = (existingEnvConfig = {}) => {
-  console.log("👛 Generating new Wallet");
+const getValidatedPassword = async () => {
+  while (true) {
+    const pass = await password({ message: "Enter a password to encrypt your private key:", mask: true });
+    const confirmation = await password({ message: "Confirm password:", mask: true });
```

Comment:
not sure we need we need a mask for passwords. If someone sees for example that my password is 4 symbols it's relatively easy to find it

Reply:
The advantage of using a mask is that the user gets better feedback while typing, right?

Reply:
> that my password is 4 symbols it's relatively easy to find it

yep, but they also need to steel your .env file :D

> The advantage of using a mask is that the user gets better feedback

Yes, this is why I added it.

But happy to reconsider

Reply:
I think maybe we hide it? 

similar to foundry workflow and also because of https://github.com/scaffold-eth/scaffold-eth-2/pull/1008#pullrequestreview-2491802349

well we don't need to follow foundry, but also due Pablos facing that issue its fine we hide it

Reply:
yeah, I think it makes sense to follow foundry and also avoiding the error on Windows.

Removed the mask for all the inputs (PK & password) d976c0a2c58d